tests/conftest.py

```diff
@@ -45,19 +38,20 @@ def tmp_products_file(tmp_path, request):
 
 
 @pytest.fixture
-def product(db_session, tmp_sqlite_db, request):
-    product_name = request.param
-    return ProductsEntity(name=product_name)
+def product(tmp_sqlite_db, request):
```

Comment:
please re-order fixture calls; first call pytest-native ones and then the ones we define